src/pages/partners/affiliate-ib/_minimum-volume.tsx

```diff
@@ -39,7 +39,7 @@ const StyledSection = styled(SectionContainer)`
         padding: 40px 0;
     }
 `
-const TitleWrapper = styled(Flex)`
+const TitleWrapper = styled.div`
```

Comment:
Any reason why flex was removed to div

Reply:
Because the texts alignment wasn't centered with flex, and i use div to follow the style used in the section above it which also use div. Also, without flex I don't need to pass extra props in titlewrapper, but still get the expected behaviour.

Reply:
why not keep the flex and add the alignment properties instead? Later this might change again and will require alignment control.